src/qml/qgismobileapp.qml

```diff
@@ -475,17 +500,26 @@ ApplicationWindow {
 
     onVertexAdded: {
       coordinateLocator.flash()
-      digitizingRubberband.model.addVertex()
+      if( stateMachine.state === 'measure' )
+        measuringRubberband.model.addVertex()
```

Comment:
here we would then avoid testing and simply have
`currentRubberand.model.addVertex()`

Reply:
I agree.